components/content/ProseCode.vue

```diff
@@ -116,6 +116,11 @@
         :deep(pre) {
             overflow: hidden;
             margin-bottom: 0;
+
+            code {
+                padding: 0;
+                color: var(--bs-white) !important;
+            }
```

Comment:
This does not loook related to the Alert change. Maybe a mistake?

If not, can you please comment?